web/pdf_scripting_manager.js

```diff
@@ -305,6 +305,27 @@ class PDFScriptingManager {
           }
           this._pdfViewer.currentScaleValue = value;
           break;
+        case "SaveAs":
+          this._eventBus.dispatch("save", { source: this });
+          break;
+        case "FirstPage":
+          this._pdfViewer.currentPageNumber = 1;
+          break;
+        case "LastPage":
+          this._pdfViewer.currentPageNumber = this._pdfViewer.pagesCount;
+          break;
+        case "NextPage":
+          this._pdfViewer.currentPageNumber = Math.min(
```

Comment:
Please just use `this._pdfViewer.nextPage()` instead, since that correctly handles things such as e.g. scroll/spread modes.